mne/forward/forward.py

```diff
@@ -642,6 +642,7 @@ def convert_forward_solution(fwd, surf_ori=False, force_fixed=False,
             x = sparse.block_diag([surf_rot] * 3)
             fwd['sol_grad']['data'] = fwd['_orig_sol_grad'] * x  # dot prod
             fwd['sol_grad']['ncol'] = 3 * fwd['nsource']
```

Comment:
For testing, read a fwd generated with `--grad` option with `surf_ori=False` and `force_fixed=False`, and read `fwd['sol_grad']['ncol']`.

Then convert it e.g. to `surf_ori=True`, and read `fwd['sol_grad']['ncol']` again.